services/core-api/app/api/mines/reports/resources/mine_report_definition_list_resource.py

```diff
@@ -0,0 +1,134 @@
+from flask_restx import Resource, reqparse
```

Comment:
This file was just renamed from `mine_report_definition_resource` to `mine_report_definition_list_resource` to match the internal naming.  

None of the code has been changed here.